tasks/broad/GermlineVariantDiscovery.wdl

```diff
@@ -174,6 +174,44 @@ task MergeVCFs {
   }
 }
 
+task Reblock {
+
+  input {
+    File gvcf
+    File gvcf_index
+    File ref_dict
+    File ref_fasta
+    File ref_fasta_index
+    String output_vcf_filename
+    String docker_image = "us.gcr.io/broad-dsde-methods/updated_reblocking@sha256:b3f98210cb5156cfe924ad017f4b834a5367b33281f5185b1a61f9e5387ed16f"
+  }
+
+  Int disk_size = ceil(size(gvcf, "GiB")) * 2 + 3
+
+  command {
+    gatk --java-options "-Xms3g -Xmx3g" \
+      ReblockGVCF \
+      -R ~{ref_fasta} \
+      -V ~{gvcf} \
+      -do-qual-approx \
+      --floor-blocks -GQB 20 -GQB 30 -GQB 40 \
+      -O ~{output_vcf_filename}
+  }
+
+  runtime {
+    memory: "3.75 GB"
+    disks: "local-disk " + disk_size + " HDD"
+    bootDiskSizeGb: 15
+    preemptible: 3
+    docker: docker_image
+  }
+
+  output {
+    File output_vcf = output_vcf_filename
+    File output_vcf_index = output_vcf_filename + ".tbi"
```

Comment:
A nit, but is it possible that some upstream task could pass a '.vcf' and NOT a '.vcf.gz'?

Reply:
The code to choose and index suffix based on whether there's a '.gz' or not is kinda gross, but if you're concerned we can hide it in one of the task WDLs.